### DISCUSSION TITLE
Bluetooth: Scanner consumption while scanning

### DISCUSSION THREAD

**Discussion by Tobi15** [zephyrproject-rtos/zephyr#54359] [Q&A]
Hi, 

I open this issue because i think i'm missing something to completely understand my power consumption measure on a Zephyr  I opened this question because I think I am missing something to fully understand my power consumption measurement on a Zephyr scanner. 

I am using a nrf52840dk with a power profiler kit 2 to measure the power consumption of my board.

Zephyr version: 3.2.99

Based on the examples provided in Zephyr for bluetooth, I created a basic application that scans continuously. I used the following scan parameters: 

```
struct bt_le_scan_param scan_param = {
  .type       = BT_HCI_LE_SCAN_PASSIVE,
  .options    = BT_LE_SCAN_OPT_NONE,
  .interval   = 48,     // 48 * 0.625 = 90 ms
  .window     = 48,  //  48 * 0.625 = 30 ms
};
```
with the following configurations : 

```
CONFIG_BT=y
CONFIG_BT_BROADCASTER=n
CONFIG_BT_OBSERVER=y
CONFIG_BT_DEBUG_LOG=n

#deep sleep
CONFIG_PM=y
CONFIG_CONSOLE=n
CONFIG_SERIAL=n
```

I got the following measurement: 

![Screenshot 2023-02-02 at 11 57 19](https://user-images.githubusercontent.com/56914662/216306448-97e8311d-4d5e-49e4-9942-56ca1e64c54a.png)

What is missing for me to understand: What are the peaks that appear every millisecond? They only appear when CONFIG_OBSERVER is on. 

Can someone help me to understand this?

Thank you very much in advance!

**Comment by kruithofa** [comment]
@cvinayak can you have a look?
@tobi15 alternatively you can ask the question at the discord channel: https://discord.com/channels/720317445772017664/733036837576376341
